src/scala/scripts/PBGenerateRequest.scala

```diff
@@ -27,7 +55,7 @@ object PBGenerateRequest {
       case "-" => Nil
       case s if s.charAt(0) == '-' => s.tail.split(':').toList //drop padding character
       case other => sys.error(s"expected a padding character of - (dash), but found: $other")
-    }) ++ List(".")
+    }) ++ List("")
```

Comment:
`::: List("")` I *think* is a bit more efficient on lists than `++` but maybe not...